Formula/pijul.rb

```diff
@@ -20,6 +20,10 @@ class Pijul < Formula
   depends_on "openssl@1.1"
 
   def install
+    # Applies a bugfix (0.20.7 -> 0.20.8) update to a dependency to fix compile.
+    # Remove with the next version.
```

Comment:
where can I see the fix for this?

Reply:
There's discussion here: https://nest.pijul.com/pijul_org/thrussh/discussions/42

Unfortunately, the [commit that fixed it](https://nest.pijul.com/pijul_org/thrussh:master/8b607fc4620ac2c69a03#ZxLW9NiJNUP) was lost because they deleted all commit history. They also don't have the source code for thrussh 0.20.x available anymore outside of crates.io.

A 0.20.8 was released to fix the issue, but the Cargo.lock still has 0.20.7 for Pijul. There has been no new release of Pijul following 0.20.8 to update this file.

Reply:
yeah, that is what I feel like (sounds like the repo is unmaintained)